src/components/CompareResults/beta/ResultsView.tsx

```diff
@@ -101,6 +104,16 @@ function ResultsView(props: ResultsViewProps) {
         );
       });
     }
+    if (framework) {
+      const frameworkId = parseInt(framework);
+      const frameworkName = frameworkMap[frameworkId as Framework['id']];
```

Comment:
The case I'm interested in is when there's a bad framework id in the URL. When that's the case, `frameworkName` will be `undefined`. It would be good to handle this case. I hope it's clearer now :)